templates/webapi/test/overview.html.ep

```diff
@@ -231,8 +235,18 @@
                         <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
                     </div>
                     <div id="add-comments-controls">
-                        <button type="submit" class="btn btn-danger">
-                            <i class="fa fa-comment"></i> Submit comment on all <%= scalar @$job_ids %> jobs
+                        <button type="button" name="bulkAction" class="btn btn-warning"
+                                value="commentJobs" id="commentJobsBtn"
+                                data-url="<%= url_for('apiv1_post_comments')->query(job_id=>$job_ids) %>"
+                                onclick="addComments(this);">
+                            <i class="fa fa-comment"></i> Comment on all <%= scalar @$job_ids %> jobs
+                        </button>
+                        <button type="button" name="bulkAction" class="btn btn-danger"
```

Comment:
The name `bulkAction` isn't used anywhere so I suppose you can just remove that attribute.